### PR TITLE
stopgap: mysql multi value index

### PR DESCRIPTION
**Pull request by eviefp**
Follow-up for https://github.com/prisma/prisma-engines/pull/3867

### CODE REVIEW

File: packages/migrate/src/__tests__/DbPull.test.ts [hunk 1]
```diff
@@ -2962,6 +2962,52 @@ describe('mysql introspection stopgaps', () => {
...
+
+
+        // introspectionSchemaVersion: NonPrisma,
+      `)
+      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+
+        // *** WARNING ***
+        // 
+        // These tables contain multi-value indices, which are not yet fully supported. Read more: https://pris.ly/d/mysql-multi-row-indextps://pris.ly/d/database-comments
```
[the remark this answers is not in the document]
Comment (eviefp) [reply packages/migrate/src/__tests__/DbPull.test.ts:3003]: nope, bad paste, fixing

**Comment by Jolg42** [comment]
General question, since we moved all warnings to the engine since https://github.com/prisma/prisma/pull/18869

What about we stop adding these tests in prisma/prisma?
Only a generic integration test would be really needed (and we have many tests already, testing warnings here) so from now only a new test in engines would be required?

**Comment by janpio** [comment]
We had discussed during the planning that we might want to combine the existing tests into one big one that covers all introspection warnings, and then a PR here would just be adding the adequate SQL and update the snapshot.

**Comment by Jolg42** [comment]
Not needed since https://github.com/prisma/prisma/pull/18869

**Pull request closed without merging**